discoverable-client/src/main/resources/static/js/hello.js

```diff
@@ -1,13 +1,15 @@
 function unsecureCall() {
     var name = $('#name').val();
     $.ajax({
-        url: "api/v1/greeting" + ((name !== undefined) ? ("?name=" + name) : "")
+        url: "api/v1/" + ((name !== undefined) ? (name + "/greeting") : "greeting")
```

Comment:
I think ```var name = $('#name').val();``` can't be undefined because it is defined in html. It may provide empty string or text from input. So when I click ```Test Rest Call``` without filling input it returns 500